mesheryctl/internal/cli/root/model/import.go

```diff
@@ -84,6 +84,7 @@ var importModelCmd = &cobra.Command{
 				}
 				err = sendToAPI(fileData, path, "file")
 				if err != nil {
+					utils.Log.Info("Error while sending file to API")
```

Comment:
to "Meshery Server", yeah? 

Or "Meshery Server API" or even better "Meshery Server API at https:///.........."